doc/whats_new.rst

```diff
@@ -196,6 +196,10 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.SquaredEpsilonInsensitive`
```

Comment:
it affects the SGD classifier code. You should point to this class not SquaredEpsilonInsensitive

let us know when you added a test or if you need help.